pkgs/stdenv/darwin/default.nix

```diff
@@ -59,11 +59,12 @@ in rec {
   stageFun = step: last: {shell             ? "${bootstrapTools}/bin/bash",
                           overrides         ? (self: super: {}),
                           extraPreHook      ? "",
+                          extraNativeBuildInputs,
                           extraBuildInputs,
```

Comment:
Isn't `extraBuildInputs` redundant? There's no cross compilation happening during the stdenv bootstrap stage anyway.